ruby/vampires.rb

```diff
@@ -0,0 +1,87 @@
+def process_employee  
+
+  # Ask how many employees will be processed in order to initiate loop #
+  puts "How many employees will be processed today?"
+  employee_count = gets.chomp.to_i
+  
+  # Create loop to process multiple employees #
+  until employee_count == 0
+
+    # Obtain applicant name, store variable and convert to boolean #
+    puts "What is your name?"
+    employee_name = gets.chomp.downcase
+      case employee_name
+      when "drake cula", "tu fang"
+        vampire_name = true
+      else
+        vampire_name = false
+      end
+
+    # Obtain applicant age, store variable and convert to integer #
+    puts "How old are you?"
+    employee_age = gets.chomp.to_i
+
+    # Obtain applicant year of birth, store variable and convert to integer #
+    puts "What year were you born?"
+    employee_year_born = gets.chomp.to_i
+    current_year = 2017
+    # Convert age variable to boolean #
+    confirm_age = current_year - employee_year_born
```

Comment:
Don't forget to account for a user who entered correct age and birth year but hasn't had their birthday yet this year.